index.html

```diff
@@ -221,6 +221,8 @@ <h3>Faculty Instructor</h3>
 Office: Beyster 3644
 <br>
 Office Hours: Monday 2:30-5pm, Wednesday 2:30-5pm
+<br>
+<i>Office hours will be virtual-only unless students are notified otherwise.</i>
```

Comment:
Since you give your office address right before the office hours, it might be helpful to clarify that the office hours are not actually in the office. I added this statement as a suggestion, but no worries if you have a better idea!